samples/nrf9160/serial_lte_modem/src/sntp/th_sntp_service.c

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#define LOG_DOMAIN time
+#define LOG_LEVEL CONFIG_THIN_LOG_LEVEL
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_DOMAIN);
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <string.h>
+#include <net/sntp.h>
+#include <time.h>
+#include "th_connect.h"
+
+#if !defined(CONFIG_THIN_SNTP_SERVER)
+#error CONFIG_THIN_SNTP_SERVER is defined
```

Comment:
The test condition says otherwise, should it be "CONFIG_THIN_SNTP_SERVER is **not** defined"?

Reply:
Yes, rephrased.